app/javascript/searchableItemList/searchableItemList.js

```diff
@@ -74,13 +75,16 @@ export function performInitialSearch({ searchOptions = {} }) {
   const responsePromise = fetchSearch('reactions', dataHash);
   return responsePromise.then((response) => {
     const reactions = response.result;
+    // FIXME: [@rhymes] the list of tags in the left column of the reading list
+    // is populated with only the tags belonging to items in the first page
```

Comment:
I haven't fixed this as it requires a larger re-design of the frontend and the backend: I think fetching the list of all tags attached to all articles in one's reading list should be a separate endpoint and a separate call from the frontend